Add Japanese localization.py

coolvitto opened the pull request.

coolvitto committed:
Add Japanese localization.py

Communist02 commented:
Thanks for the localization in Japanese! I updated the installer and added localization to it too

coolvitto commented:
Hello,
Thank you so much for the quick update.

coolvitto
________________________________
差出人: Mazur Denis ***@***.***>
送信日時: 2025年3月18日 19:06
宛先: Communist02/AnimePlayer ***@***.***>
CC: coolvitto ***@***.***>; Author ***@***.***>
件名: Re: [Communist02/AnimePlayer] Add Japanese localization.py (PR #11)


Thanks for the localization in Japanese! I updated the installer and added localization to it too

—
Reply to this email directly, view it on GitHub<https://github.com/Communist02/AnimePlayer/pull/11#issuecomment-2732509219>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BGH4IKZZFIIK5VNLA2I23UT2U7V2DAVCNFSM6AAAAABZHLMK4CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMZSGUYDSMRRHE>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

[Communist02]Communist02 left a comment (Communist02/AnimePlayer#11)<https://github.com/Communist02/AnimePlayer/pull/11#issuecomment-2732509219>

Thanks for the localization in Japanese! I updated the installer and added localization to it too

—
Reply to this email directly, view it on GitHub<https://github.com/Communist02/AnimePlayer/pull/11#issuecomment-2732509219>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BGH4IKZZFIIK5VNLA2I23UT2U7V2DAVCNFSM6AAAAABZHLMK4CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMZSGUYDSMRRHE>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
